scripts/applyPatches.sh

```diff
@@ -4,8 +4,8 @@
 PS1="$"
 basedir="$(cd "$1" && pwd -P)"
 workdir="$basedir/work"
-gpgsign="$(git config commit.gpgsign || echo "false")"
-applycmd="git am --3way --ignore-whitespace"
+applycmd="$gitcmd am --3way --ignore-whitespace"
+gitcmd="git -c commit.gpgsign=false"
```

Comment:
Looks like this needs to be above applycmd in order for macOS and Travis-CI to build.